examples/dmd/local_dw_csv.cpp

```diff
@@ -0,0 +1,889 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Compile with: make local_dw_csv
+//
+// Generate CSV databse on heat conduction with: heat_conduction_csv.sh
```

Comment:
Does this example still work?

Reply:
Right now the new scripts `*_dw_csv` only apply to pore collapse data, as the indicator is specific in different applications. Right now, there are no examples of command line options that can test the new capabilities of the new scripts `*_dw_csv`. Examples will be added In a future PR.